geopandas/plotting.py

```diff
@@ -114,6 +114,15 @@ def plot_series(s, cmap='Set1', ax=None, linewidth=1.0, figsize=None, **color_kw
 
         matplotlib axes instance
     """
+    if 'colormap' in color_kwds:
+        warnings.warn("'colormap' is deprecated, please use 'cmap' instead "
+                      "(for consistency with matplotlib)", FutureWarning)
+        cmap = color_kwds.pop('colormap')
```

Comment:
Should we guard against the (admittedly perverse) case in which the user specifies both `cmap` and `colormap`?